### ISSUE TITLE
"hamburger" style settings menu in Linux

### ISSUE THREAD

**Issue by brittyazel** [nylas/N1#1250]
I'm not sure if this has been requested or not before, but would it be possible to add the "hamburger" style drop down settings menu, found in the Windows build of N1, to the Linux version instead of the horizontal menu across the top?

I feel like the horizontal menu makes the UI look out of place in all desktop environments besides Unity, and that the "hamburger" style dropdown is much cleaner looking visually, and would be much more aligned with the design direction of the project. Further, many Gnome and KDE apps (and 3rd party apps such as Chrome and Firefox) already adopt a similar single drop-down menu paradigm, so it would not be out of place in Linux for N1 to carry this feature as well.

Lastly, the horizontal menu just looks bad, which is in stark contrast to the gorgeous look of the rest of the system. And the dark theme does not work at all with the horizontal menu, thus making the light theme the only visually appealing option.

**Comment by mbilker** [comment]
To style the menu bar, you need to set your GTK2 and GTK3 themes to a dark theme. Chromium derives its menu bar theme from GTK.

Regarding the menu bar, the menu bar is present on MATE applications and I prefer it more than GNOME and KDE, which are moving in a direction I do not prefer. I use the menu bar a lot for development and periodic checking of the Activity Window to ensure an action and its corresponding API request succeeds.

**Comment by brittyazel** [comment]
I agree that more options are always better. Is it possible to then under view add the option to "show menubar" or "show hamburger menu". That way it can be left up to the user to decide which way they prefer. Considering the Windows version already adopts the hamburger style, it doesn't seem that unreasonable to give Linux users the same menu style

**Comment by mank319** [comment]
I totally agree with @brittyazel!
A menu bar makes a "simple application" such as a mail client look cluttered and outdated on most modern Linux desktops (Gnome, Pantheon, Budgie, even KDE...). I do see the advantage of a menu bar in professional tools with many, deeply structured menus such as IDEs, Photo Editing Tools or even Office Suites, but in my opinion "modern email clients" do not fall into that category.
